src/main/java/com/privacylogistics/FF3Cipher.java

```diff
@@ -156,8 +156,8 @@ public String encrypt(String plaintext, byte[] tweak) throws BadPaddingException
         int v = n - u;
 
         // Split the message
-        String A = plaintext.substring(0, u);
-        String B = plaintext.substring(u);
+        char[] A = plaintext.substring(0, u).toCharArray();
```

Comment:
As of Java 7, substring creates a new String from the character array.  What about using[getChars(int srcBegin,
 int srcEnd, char[] dst, int dstBegin)]
(https://docs.oracle.com/en/java/javase/17/docs/api/java.base/java/lang/String.html#getChars(int,int,char%5B%5D,int))?

Reply:
Did it help the benchmark any?